drivers/flash/Kconfig.it8xxx2

```diff
@@ -0,0 +1,17 @@
+# FLASH configuration options
+
+# Copyright (c) 2021 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FLASH_ITE_IT8XXX2
+	bool "ITE IT8XXX2 flash driver"
+	depends on SOC_FAMILY_RISCV_ITE
+	select SOC_FLASH_RAMCODE_SECTION
+	help
+	  Enable the ITE it8xxx2 series internal flash driver.
```

Comment:
Can you add at least 3 lines of help. What features does the driver support? How large is the flash?

Reply:
Done

Reply:
nit: Can this be reworded?  Possibly as...

The flash driver includes support for read, write and
erase flash operations.  It also supports protection.
The it8xxx2 flash size is 1M byte.

Reply:
Done. Thanks Denis.